lambdas/issuecredential/src/main/java/uk/gov/di/ipv/cri/passport/issuecredential/service/VerifiableCredentialService.java

```diff
@@ -0,0 +1,130 @@
+package uk.gov.di.ipv.cri.passport.issuecredential.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.nimbusds.jose.JOSEException;
+import com.nimbusds.jwt.SignedJWT;
+import uk.gov.di.ipv.cri.common.library.annotations.ExcludeFromGeneratedCoverageReport;
+import uk.gov.di.ipv.cri.common.library.domain.personidentity.BirthDate;
+import uk.gov.di.ipv.cri.common.library.domain.personidentity.PersonIdentityDetailed;
+import uk.gov.di.ipv.cri.common.library.util.KMSSigner;
+import uk.gov.di.ipv.cri.common.library.util.SignedJWTFactory;
+import uk.gov.di.ipv.cri.common.library.util.VerifiableCredentialClaimsSetBuilder;
+import uk.gov.di.ipv.cri.passport.issuecredential.domain.verifiablecredential.Passport;
+import uk.gov.di.ipv.cri.passport.issuecredential.util.EvidenceHelper;
+import uk.gov.di.ipv.cri.passport.library.persistence.DocumentCheckResultItem;
+import uk.gov.di.ipv.cri.passport.library.service.PassportConfigurationService;
+import uk.gov.di.ipv.cri.passport.library.service.ServiceFactory;
+
+import java.time.Clock;
+import java.time.format.DateTimeFormatter;
+import java.time.temporal.ChronoUnit;
+import java.util.List;
+import java.util.Map;
+
+import static uk.gov.di.ipv.cri.passport.issuecredential.domain.VerifiableCredentialConstants.VC_BIRTHDATE_KEY;
+import static uk.gov.di.ipv.cri.passport.issuecredential.domain.VerifiableCredentialConstants.VC_CREDENTIAL_TYPE_ICC;
+import static uk.gov.di.ipv.cri.passport.issuecredential.domain.VerifiableCredentialConstants.VC_NAME_KEY;
+import static uk.gov.di.ipv.cri.passport.issuecredential.domain.VerifiableCredentialConstants.VC_PASSPORT_KEY;
+import static uk.gov.di.ipv.cri.passport.library.config.ParameterStoreParameters.MAX_JWT_TTL_UNIT;
+
+public class VerifiableCredentialService {
+    public static final String ICAO_ISSUER_CODE = "GBR";
```

Comment:
Could we name this UK_ICAO_ISSUER_CODE I have concerns we will forget its only setup for the UK